bin/align_rts_from_easypqp.py

```diff
@@ -82,6 +82,7 @@ def compute_MST(libs,min_overlap):
 
 ### TODO: Finish
```

Comment:
can you document each function? add that you are the author somewhere?

Reply:
I added a bit more description, hope it got better..